omero/sysadmins/server-upgrade.rst

```diff
@@ -116,6 +116,14 @@ precautionary checks also done by the actual upgrade script.
     (1 row)
 
 
+.. warning::
+
+   The :file:`sql/psql/OMERO5.4__0/OMERO5.3__1-precheck.sql` script
+   referenced by the above :program:`psql` command assumes a planned
+   upgrade from OMERO 5.3.4. If you are instead currently running OMERO
+   5.3.0 to 5.3.3 then you perform the precheck by using the above
+   command with :file:`sql/psql/OMERO5.4__0/OMERO5.3__0-precheck.sql`.
+
```

Comment:
Maybe add a note that the script checks if the DB is "corrupted"
Actually we do not offer a way to fix that issue